src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = function(numA) {
+    count++;
+
+    if (count < 3 || count % 2 === 1) {
+      return (numB) => numA + numB;
+    };
+
+    return () => 'Bzzz... Error!';
```

Comment:
Why returning two functions here? Just return one and choose what value to return inside it